docs/manual/installation/install-contao.de.md

```diff
@@ -41,8 +41,9 @@ werden.
 
 ### Datenbanktabellen aktualisieren
 
-Sobald der Contao Manager alle Pakete installiert hat, musst du das [Contao-Installtool](../contao-installtool/)
-aufrufen um die Datenbank zu aktualisieren.
+Sobald der Contao Manager alle Pakete installiert hat, muss die Datenbank aktualisiert werden. Dazu muss das
+[Contao-Installtool](../contao-installtool/) gestartet werden (Contao 4.13 LTS) bzw. die benötigten Informationen direkt im Contao Manager 
+erfasst werden (Contao 5.x).
```

Comment:
This does not seem to be correct. There is nothing that you need to "erfassen" in the Contao Install Tool? Also you do not need the Install Tool at all. The Contao Manager will just prompt you to update the database directly.

Reply:
Ok, changed

Reply:
I had taken the sentence from 
https://docs.contao.org/manual/de/installation/quickstart/#datenbanktabellen-aktualisieren. 
Should we change this as well?

Reply:
I think so, yes.